README.md

```diff
@@ -1,5 +1,29 @@
-# Momento client-types-php
+# client-sdk-php
 
-:warning: Pre-Alpha :warning:
+:warning: Experimental SDK :warning:
+
+PHP SDK for Momento is experimental and under active development. There could be non-backward compatible changes or
+removal in the future. Please be aware that you may need to update your source code with the current version of the SDK
+when its version gets upgraded.
+---
+
+<br />
+PHP SDK for Momento, a serverless cache that automatically scales without any of the operational overhead required by
+traditional caching solutions.
+
+<br/>
+
+# Getting Started :running:
+
+## Requirements
+
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+- At least PHP 7
+- The grpc PHP extension. See the [gRPC docs](https://github.com/grpc/grpc/blob/v1.46.3/src/php/README.md) section on installing the extension.
+
+## Using Momento
+
+Run `composer install` from the root of the repository to install the necessary prerequisites.
```

Comment:
does `composer` come with the PHP install?  or do we need to tell folks how to install that?

Reply:
Great call, I'll add a link to instructions.